components/Map/index.js

```diff
@@ -0,0 +1,144 @@
+    long: 7.432,
+    description:
+      "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.",
+  },
+  {
+    id: 4,
+    name: "Cars and Coffe",
+    lat: 48.54,
+    long: 8.432,
+    description:
+      "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.",
+  },
+];
+export default function Map() {
+  const [selectedEvent, setSelectedEvent] = useState(false);
+  return (
+    <>
+      <StyledPageHeadline>Find events for your tour!</StyledPageHeadline>
+      <StyledMapContainer center={[48.521, 9.057]} zoom={8} scrollWheelZoom>
+        <TileLayer
+          attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+          url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+        />
+        {markers.map((marker) => {
+          return (
+            <Marker
+              key={marker.id}
+              position={[marker.lat, marker.long]}
+              icon={eventIcon}
+            >
+              <Popup>
+                <h2>{marker.name}</h2>
+                <button
+                  type="button"
+                  onClick={() => {
+                    setSelectedEvent(
+                      markers.find((mark) => mark.id === marker.id)
+                    );
+                  }}
+                >
+                  See Details
+                </button>
+              </Popup>
+            </Marker>
+          );
+        })}
+        <LocationMarker />
+      </StyledMapContainer>
+      {selectedEvent ? (
+        <StyledDetails>
+          <StyledHeadline>{selectedEvent.name}</StyledHeadline>
+          <StyledDescription>{selectedEvent.description}</StyledDescription>
+          <StyledLink
+            href={`https://maps.google.com/?q=${selectedEvent.lat},${selectedEvent.long}`}
+          >
+            Open on Google Maps
+          </StyledLink>
+        </StyledDetails>
+      ) : null}
```

Comment:
you can work with the logical and here:

```suggestion
      {selectedEvent && (
        <StyledDetails>
          <StyledHeadline>{selectedEvent.name}</StyledHeadline>
          <StyledDescription>{selectedEvent.description}</StyledDescription>
          <StyledLink
            href={`https://maps.google.com/?q=${selectedEvent.lat},${selectedEvent.long}`}
          >
            Open on Google Maps
          </StyledLink>
        </StyledDetails>
      )}
```

Reply:
Done